hw/apps.js

```diff
@@ -0,0 +1,102 @@
+
+///1
+let body= document.querySelector('body');
+body.style.fontFamily='Arial, sans-serif';
+// body.style.color='green';
+
+// document.querySelectorAll('span');
+// document.querySelectorAll('span')[0];
+// document.getElementsByTagName('span');
+
+// document.getElementsByTagName('span')[0];
+//2
+let nickname=document.querySelector('#nickname');
+nickname.innerText="Ashwag";
+
+let favorite=document.querySelector('#favorite');
+favorite.innerText="cat";
+
+let hometown=document.querySelector('#hometown');
+hometown.innerText="Riyadh";
+
+
+///2
+
+document.getElementsByClassName('profilePicture');
+
+
+//3
+var listitems = document.getElementsByTagName("li");
+for (let i = 0; i < listitems.length; i++)
+{
+ document.createElement('li.listitem')[i];
```

Comment:
This creates an element, what the question wanted you to do is give the li elements a class name, a simple way of doing that would be using `setAttribute()` or `.classList` or `.className`